Makefile

```diff
@@ -32,12 +29,10 @@ include $(HACK_DIR)/tools.mk
 .PHONY: tidy
 tidy:
 	@env GO111MODULE=on go mod tidy
-	@GARDENER_HACK_DIR=$(GARDENER_HACK_DIR) bash $(HACK_DIR)/update-github-templates.sh
-	@cp $(GARDENER_HACK_DIR)/cherry-pick-pull.sh $(HACK_DIR)/cherry-pick-pull.sh && chmod +xw $(HACK_DIR)/cherry-pick-pull.sh
 
 .PHONY: clean
 clean:
-	@bash $(GARDENER_HACK_DIR)/clean.sh ./api/... ./internal/...
+	@$(HACK_DIR)/clean.sh ./api/... ./internal/...
```

Comment:
In most of the make targets we're calling the shell scripts by prepending the script filename with the `bash` command - but in the proposed changes this convention wasn't followed.

We could possibly make all script invocations through the explicit `bash` command to ensure the targets work without any problem in incompatible shells like `fish` as well.

Reply:
`fish` is not a posix compliant shell. https://wiki.gentoo.org/wiki/Fish also as the following warning:
```
fish should not be set as the system shell by linking in /bin/sh, this could result in an inoperable system.
```
even upstream projects like k8s (https://github.com/kubernetes/kubernetes/blob/master/build/root/Makefile), etcd (https://github.com/etcd-io/etcd/blob/main/Makefile), our own gardener (https://github.com/gardener/gardener/blob/master/Makefile) also do not care about invoking commands via `bash`

Reply:
All the make targets I've tested indeed work on shells like `fish` because all our scripts indicate the file is a bash script through the `#!/bin/bash` comment at the top.

Since this is the case, it would be cleaner to not invoke bash explicitly in make targets since it is redundant?

Reply:
yes i would also second that.